src/Common/ProviderAggregator.php

```diff
@@ -174,7 +166,7 @@ public function getProviders(): array
      *
      * @throws ProviderNotRegistered
      */
-    private static function getProvider($query, array $providers, Provider $currentProvider = null): Provider
+    private static function getProvider($query, array $providers, Provider $currentProvider = null): Provider // @phpstan-ignore-line
```

Comment:
maybe use phpstan baseline, if we could fix this later instead of inlining ignoring here?

Reply:
Done in b3ce75f5759678df5535726c4daf5920d979dec4